test/helpers/cons.go

```diff
@@ -303,6 +303,8 @@ var ciliumCLICommands = map[string]string{
 	"cilium policy get":                     "policy_get.txt",
 	"cilium status --all-controllers":       "status.txt",
 	"cilium kvstore get cilium --recursive": "kvstore_get.txt",
+
+	"hubble observe --since 4h -o json": "hubble_observe.txt",
```

Comment:
nit: Would you mind adding a FIXME note to use https://github.com/cilium/hubble/issues/253 once its implemented? `hubble observe --all -o json` sounds much more sensible :)

Reply:
I'll add a note in the linked issue, so the person implementing or reviewing the `--all` will have a chance to issue a PR. Adding the FIXME cons.go will probably never be seen by most devs (also, the FIXME will would retrigger CI :P)